niyodusengaclement/taskforce-todo-backend
# ft():add docker in a project

#### Description of changes made
---

> - Add Dockerfile
> - Add docker-compose.yml

#### Type of change
---

> **Please select the relevant options**



#### How should this be manually tested?
---

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/niyodusengaclement/taskforce-todo-backend.git), and `cd [the project name]`
- Check out this branch by `git checkout [the branch for this pr]`
- Run ` docker build .` to Build Docker Image
- Run `docker-compose up` to Start App with Docker
- .....

#### Checklist:



#### Screenshots (if appropriate)
